easybuild/easyconfigs/w/whisper-ctranslate2/whisper-ctranslate2-0.5.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'whisper-ctranslate2'
+version = '0.5.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/Softcatala/whisper-ctranslate2/'
+description = "Whisper command line client compatible with original OpenAI client based on CTranslate2."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('tqdm', '4.66.1'),
+    ('tokenizers', '0.15.2'),
+    ('CTranslate2', '4.5.0', versionsuffix),
+    ('PyAV', '11.0.0'),
+    ('ONNX-Runtime', '1.19.2', versionsuffix),
+    ('cffi', '1.15.1'),
+    ('PortAudio', '19.7.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('huggingface-hub', '0.26.5', {
+        'sources': ['huggingface_hub-%(version)s.tar.gz'],
+        'checksums': ['1008bd18f60bfb65e8dbc0a97249beeeaa8c99d3c2fa649354df9fa5a13ed83b'],
+    }),
+    ('sounddevice', '0.5.1', {
+        'checksums': ['09ca991daeda8ce4be9ac91e15a9a81c8f81efa6b695a348c9171ea0c16cb041'],
+    }),
+    ('faster-whisper', '1.1.1', {
+        'checksums': ['50d27571970c1be0c2b2680a2593d5d12f9f5d2f10484f242a1afbe7cb946604'],
+    }),
+    (name, version, {
+        'modulename': False,
+        'source_urls': ['https://github.com/Softcatala/whisper-ctranslate2/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'checksums': ['8a7c7b232086b8bfa7022f3a3b3f95c8c5175c33663cf8d3fd840539f2be9a49'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [f'bin/{name}'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [f"{name} --help"]
```

Comment:
@pavelToman Don't use f-strings just yet please, strictly speaking EasyBuild 4.x still supports Python 2.7...

Very soon, as soon as we collapse the `5.0.x` branches into `develop` shortly before finally releasing EasyBuild 5.0, this will be fine